.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature Request
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I would remove the "Pleass describe" as this is part of what you want the submitter do and which goes in the next line.

Reply:
Done as desired